public/app/features/variables/query/QueryVariableRefreshSelect.tsx

```diff
@@ -14,14 +15,15 @@ const REFRESH_OPTIONS = [
 ];
 
 export function QueryVariableRefreshSelect({ onChange, refresh }: PropsWithChildren<Props>) {
+  const { width } = useWindowSize();
   const value = useMemo(
     () => REFRESH_OPTIONS.find((o) => o.value === refresh)?.value ?? REFRESH_OPTIONS[0].value,
     [refresh]
   );
 
   return (
     <Field label="Refresh" description="When to update the values of this variable">
-      <RadioButtonGroup options={REFRESH_OPTIONS} onChange={onChange} value={value} />
+      <RadioButtonGroup options={REFRESH_OPTIONS} onChange={onChange} value={value} size={width < 375 ? 'sm' : 'md'} />
```

Comment:
I don't think this is a good solution (not worth it, and just makes code messy, and not a pattern we use anywhere else )

Reply:
beside the radio button fits on small screens. I would change the wording instead "on time range change" to just "On time change" 
<img width="381" alt="Screenshot 2023-07-24 at 20 54 23" src="https://github.com/grafana/grafana/assets/10999/640212d8-204e-4310-989b-cb66fee5a523">

Reply:
what do you suggest to make it fit a mobile screen?

Reply:
I'm also wondering how responsive we should be since a 320px mobile device is not a common one anymore. 
Changing the wording keeps generating some problems, since there's a minimal horizontal scrolling and no padding. Shouldn't we decrease the padding a little bit more, in this case? 
![image](https://github.com/grafana/grafana/assets/11707172/f554a95d-a84b-4f69-be09-1acd7c4884a1)